.gitignore

```diff
@@ -16,3 +16,5 @@
 /polyfills/Intl/~locale
 
 polyfills/Promise/polyfill.js
+
+polyfills/atob/polyfill.js
```

Comment:
I'd be tempted to not even have the polyfills copied to this location and instead have the coed in `lib` deal with the third party install method directly.